lib/cf-socket.c

```diff
@@ -1828,6 +1827,36 @@ CURLcode Curl_conn_tcp_listen_set(struct Curl_easy *data,
   return result;
 }
 
+static void set_accepted_remote_ip(struct Curl_cfilter *cf,
+                                   struct Curl_easy *data)
+{
+  struct cf_socket_ctx *ctx = cf->ctx;
+#ifdef HAVE_GETPEERNAME
+  char buffer[STRERROR_LEN];
+  struct Curl_sockaddr_storage ssrem;
+  curl_socklen_t plen;
+
```

Comment:
Minor thing, but maybe have:
```
  ctx->r_ip[0] = 0;
  ctx->r_port = 0;
```
...here to set some default values in case there's an error.

Reply:
Good suggestion. Added to the PR.